packages/document/rspress.config.ts

```diff
@@ -169,4 +169,8 @@ export default defineConfig({
       },
     },
   },
+  mediumZoom: {
+    // Select all images that are NOT descendants of an anchor
+    selector: '.rspress-doc img:not(a img)',
```

Comment:
This is required, otherwise, those buttons are acting like images that can be zoomed in.

I believe this is relatively safe as we probably never want to zoom on an image within an anchor, but please let me know if this is not safe!